### PR TITLE
[WIP] Add a test for decimal32 to_arrow in libcudf

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

File: cpp/src/interop/to_arrow.cu [hunk 1]
```diff
@@ -182,6 +182,51 @@ std::shared_ptr<arrow::Array> dispatch_to_arrow::operator()<numeric::decimal64>(
...
+  rmm::cuda_stream_view stream)
+{
+  using DeviceType                = int32_t;
+  size_type const BIT_WIDTH_RATIO = 4;  // Array::Type:type::DECIMAL (128) / int32_t
+
+  rmm::device_uvector<DeviceType> buf(input.size() * BIT_WIDTH_RATIO, stream);
+
+  auto count = thrust::make_counting_iterator(0);
+
+  thrust::for_each(count,
```
Comment (jrhemstad) [inline cpp/src/interop/to_arrow.cu:200]: missing exec

Comment (shwina) [reply cpp/src/interop/to_arrow.cu:200]: Oooh, it looks like we were missing it for the 64-bit case too :D Does this mean we are computing it on the host?

Comment (shwina) [reply cpp/src/interop/to_arrow.cu:200]: cc: @codereport to share the blame with me

**Comment by galipremsagar** [comment]
Moving this PR to `21.12`

**Comment by shwina** [comment]
Closing this as it has gone stale.

**Pull request closed without merging**